java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -90,18 +90,42 @@ public interface ListBaseCommands {
      *     If <code>key</code> does not exist an empty array will be returned.<br>
      * @example
      *     <pre>{@code
-     * String[] payload = lient.lrange("my_list", 0, 2).get()
-     * assert payload.equals(new String[] {"value1", "value2", "value3"})
+     * String[] payload = lient.lrange("my_list", 0, 2).get();
+     * assert payload.equals(new String[] {"value1", "value2", "value3"});
      *
-     * String[] payload = client.lrange("my_list", -2, -1).get()
-     * assert payload.equals(new String[] {"value2", "value3"})
+     * String[] payload = client.lrange("my_list", -2, -1).get();
+     * assert payload.equals(new String[] {"value2", "value3"});
      *
-     * String[] payload = client.lrange("non_exiting_key", 0, 2).get()
-     * assert payload.equals(new String[] {})
+     * String[] payload = client.lrange("non_exiting_key", 0, 2).get();
+     * assert payload.equals(new String[] {});
      * }</pre>
      */
     CompletableFuture<String[]> lrange(String key, long start, long end);
 
+    /**
+     * Returns the element at <code>index</code> in the list stored at <code>key</code>.<br>
+     * The index is zero-based, so 0 means the first element, 1 the second element and so on. Negative
```

Comment:
wrap numbers and `null` with `code`

Reply:
I don't think it makes sense to wrap the numbers here they just tell about indexing.

Reply:
It will be consistend with other commands, see `ttl` for example